scripts/collect-windows-logs.ps1

```diff
@@ -86,6 +86,18 @@ else {
   Write-Host "Containerd hyperv logs not avalaible"
 }
 
+# log containerd containers (this is done for docker via networking collectlogs.ps1)
+$res = Get-Command ctr.exe -ErrorAction SilentlyContinue
+if ($res)
+{ 
+  & ctr.exe -n k8s.io c ls > "$ENV:TEMP\$timeStamp-containerd-containers.txt"
+  & ctr.exe -n k8s.io t ls > "$ENV:TEMP\$timeStamp-containerd-tasks.txt"
+  & wpr.exe -stop "$ENV:TEMP\$timeStamp-containerd.etl"

```

Comment:
New to wpr, just curious, where do we start the trace?

Reply:
We aren't starting the trace.  Someone would need to start the trace, repo the issue and then run this script to collect all the logs.  If there is no trace running this would report that, and continue on.  We are going to try a different approach all together here so this will change.

Reply:
We moved to using nssm to enable consistent logging story across our components